products/compute/inspec.yaml

```diff
@@ -82,7 +82,12 @@ overrides: !ruby/object:Overrides::ResourceOverrides
   TargetHttpsProxy: !ruby/object:Overrides::Inspec::ResourceOverride
     exclude: true
   TargetPool: !ruby/object:Overrides::Inspec::ResourceOverride
-    exclude: true
+    additional_functions: |
```

Comment:
Could you put this in a separate file? If it's a separate .rb file, IDEs/editors will treat the file as normal Ruby code. We lose all of that if it's Ruby embedded in YAML.